js/playerList.js

```diff
@@ -0,0 +1,108 @@
+const buttonAdd = document.querySelector("#user-add-game");
+const draft = document.querySelector("#draft");
+
+buttonAdd.addEventListener("click", function (e) {
+  getCheckedCheckBoxes();
+});
+
+draft.addEventListener("click", function (e) {
+  createPlay();
+});
+
+//Добавление игроков в список
+function getCheckedCheckBoxes() {
+  let checkboxes = document.getElementsByClassName("checkbox");
+  let checkboxesChecked = [];
+  for (let i = 0; i < checkboxes.length; i++) {
+    if (checkboxes[i].checked) {
+      checkboxesChecked.push(checkboxes[i].value);
+    }
+  }
+  let i = -1;
+  document.querySelector(".test").innerHTML += checkboxesChecked
+    .map(() => {
+      i++;
+      return `<tr id="${i + 1}">
+            <td width="50px">${i + 1}</td>
+            <td width="550px"><p class="pl">${checkboxesChecked[i]}</p></td>
+            <td width="50px"><button class="remove" onClick="removeRow(${
+              i + 1
+            })" value="${i + 1}">✖</button></td>
+        </tr>`;
+    })
+    .join("");
+}
+
+//Удаление объекта
+function removeRow(id) {
+  let description = document.getElementById(id);
+  description.remove();
+}
+
+function par() {
+  let players = document.getElementsByClassName("tur");
+  let playersChecked = [];
+  for (let i = 0; i < players.length; i+=4) {
+    playersChecked[i/4] = {
+      fioPOne: players[i].innerHTML,
+      fioPTwo: players[i + 1].innerHTML,
+      RP: players[i + 2].value + players[i + 3].value,
```

Comment:
Вот вместо rp сделал бы такую штук
```suggestion
      won: players[i + 2].value,
      lost: players[i + 3]. value,
```
Потому что работать так со строчкой как-то не очень читаемо